src/ResourceManager/Resources/Commands.ResourceManager/Cmdlets/Implementation/Policy/NewAzurePolicyAssignment.cs

```diff
@@ -54,6 +61,19 @@ public class NewAzurePolicyAssignmentCmdlet : PolicyAssignmentCmdletBase
         public PSObject PolicyDefinition { get; set; }
 
         /// <summary>
+        /// Gets or sets the policy assignment policy parameter object.
+        /// </summary>
+        [Parameter(Mandatory = false, ValueFromPipelineByPropertyName = true, HelpMessage = "The pollicy parameter object.")]
+        public Hashtable PolicyParameterObject { get; set; }
```

Comment:
We should have validation so that user cannot specify both PolicyParameterObject and PolicyParameters.
This can be implemented using parameter sets.